### PR TITLE
fix(app menu): panel nub positioning

### PR DESCRIPTION
**Pull request by dmy7r0**
## Description

Update the styling (color) and positioning of the collapsible panel nubs

### CODE REVIEW

File: src/platform-implementation-js/dom-driver/gmail/gmail-driver/add-app-menu-item.ts [hunk 1]
```diff
@@ -16,8 +16,8 @@ export async function addAppMenuItem(
   try {
     if (!appMenuInjectionContainer || !gmailAppMenuItemView.element) return;
 
-    const siblingElement = menuItemDescriptor.insertIndex
-      ? appMenuInjectionContainer.childNodes[menuItemDescriptor.insertIndex] ??
+    const siblingElement = Number.isInteger(menuItemDescriptor.insertIndex)
```
Comment (wegry) [inline src/platform-implementation-js/dom-driver/gmail/gmail-driver/add-app-menu-item.ts:19]: Can't say I've seen this method before. Nice.

File: src/platform-implementation-js/dom-driver/gmail/gmail-driver/add-collapsible-panel.ts [hunk 2]
```diff
@@ -24,7 +25,22 @@ export async function addCollapsiblePanel(
 
   if (!appMenu) return;
 
-  appMenu.insertAdjacentElement('afterend', collapsiblePanelView.element);
+  const panelNodes =
+    injectionContainer.querySelectorAll<HTMLElement>('.aqn') ?? [];
```
Comment (wegry) [inline src/platform-implementation-js/dom-driver/gmail/gmail-driver/add-collapsible-panel.ts:29]: Would we want to add a static constant or use `CollapsiblePanelView.elementSelectors.NATIVE` here?

**Review by wegry — APPROVED** [review]
Seems to work pretty well with just adding menu items.

This is sort of a contrived example edge case, but we might want to adjust nub position when app menu items are removed.

With the following diff:

```diff
diff --git a/examples/app-menu/content.js b/examples/app-menu/content.js
index 933606cc..577f5c77 100644
--- a/examples/app-menu/content.js
+++ b/examples/app-menu/content.js
@@ -111,4 +111,8 @@ InboxSDK.load(2, 'app-menu').then(async (sdk) => {
       name: 'Nav Item 1',
       onClick: () => sdk.Router.goto('custom-route-3'),
     })
+
+  setTimeout(() => {
+    customItem1.remove();
+  }, 10_000);
 });
```

Tooltip position doesn't appear to be updating. Probably not worth blocking this going in though.

https://user-images.githubusercontent.com/5156873/222509249-520c83c3-00a8-4bf0-b2ca-e0ee3bf67c4e.mov

File: src/platform-implementation-js/dom-driver/gmail/gmail-driver/add-collapsible-panel.ts (hunk 2 above)
Comment (dmy7r0) [reply src/platform-implementation-js/dom-driver/gmail/gmail-driver/add-collapsible-panel.ts:29]: Nice

**Pull request merged**